lerna.json

```diff
@@ -1,7 +1,7 @@
 {
   "packages": [
     "infrastructure",
-    "applications/**"
+    "packages/api"
```

Comment:
```suggestion
    "packages/**"
```

Is there a specific reason we'd want to enumerate every package? l am concerned it'd be an easy thing to forget in the future

Reply:
No.  I agree could be easy to forget.  But I found plenty of other places where we'd have to specify resources in a package.